docs/core/versions/selection.md

```diff
@@ -141,11 +141,11 @@ The roll-forward behavior for an application can be configured in four different
 
 ### Precedence
 
-Roll forward behavior is considered in the following order:
+Roll forward behavior is set by the following order when your app is run, higher numbered items taking precedence over lower numbered items:
```

Comment:
Since you've built precedence info into the step descriptions themselves, you could also shorten the intro:
```suggestion
Roll forward behavior is determined by the following procedure when your app is run:
```